docs/release/release_notes.md

```diff
@@ -32,6 +32,16 @@ The DAOS 2.0.1 release contains the following updates on top of DAOS 2.0.0:
 
 - `spdk` has been updated from 21.07-8 to 21.07-11 (minor fixes only).
 
+## Known Issues and limitations
+
+During SOAK testing of DAOS 2.0.1,
+some test jobs have failed with DER\_CSUM(-2021) checksum errors.
+This sighting is currently being investigated to determine if this is
+caused by a client-side issue with the checksum verification code,
+triggered by a hardware media error in the testing environment, or is
+a bug in the server code that may potentially cause a data corruption.
+See [DAOS-9725](https://daosio.atlassian.net/browse/DAOS-9725).
+
```

Comment:
@johannlombardi So you wanted us to restore the known issues that were in 2.0 but still persist in 2.0.1, correct?

Reply:
Right, there might be some other bugs we want to mention there.

Reply:
done